edx_exams/apps/api/v1/views.py

```diff
@@ -400,10 +414,28 @@ class ExamAttemptView(ExamsAPIView):
 
     PUT Response Values
         {'exam_attempt_id': <attempt_id>}: The attempt id of the attempt being updated
+        
+    **Exceptions**
+        * HTTP_400_BAD_REQUEST
+        * HTTP_403_FORBIDDEN
     """
 
     authentication_classes = (JwtAuthentication,)
 
+    def get(self, request, attempt_id):
+        # TODO, maybe: Only allow authenticated users with the right role to view other's exam attempts
```

Comment:
This should definitely be added (if we are filtering by attempt ID). A user should only be able to request data on their own attempts, unless they have global staff access